src/components/calculator.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/self-closing-comp */
+/* eslint-disable react/button-has-type */
+/* eslint-disable jsx-quotes */
+import React from 'react';
+
+import './Calculator.css';
+
```

Comment:
- [ ] Let's try not to disable Linter checks as they help us write better and more professional code. Disabling Linters defies its purpose. Please fix the error 👍🏽
- [ ] The import for the calculator CSS file from the components directory is in uppercase but the file is actually in lowercase and for that reason, the wrong import is being done and the app is not starting or loading the css. Kindly work on that 👍🏽

Reply:
Thanks for your suggestions. 

I have a problem every time I push the code, but it is working on my computer and other repositories. I checked for errors on my vs - npx eslint .  and no mistake shows.

I have also run npm start - and the server is running without any error. (see screenshot below)

Please advise or help me with the way forward.

![ ESlint error on pr - Screenshot 2023-01-11 at 2 33 27 PM](https://user-images.githubusercontent.com/106531695/211796342-df1fe26e-e2f6-4236-99b7-fcef4319b2ee.png)


![Screenshot 2023-01-11 at 2 28 42 PM](https://user-images.githubusercontent.com/106531695/211795648-8302e420-83bb-4f5d-9a44-315f47e6d666.png)

![Screenshot 2023-01-11 at 2 30 21 PM](https://user-images.githubusercontent.com/106531695/211795637-70e93510-345f-4190-988d-d3ae5c5fc741.png)

![Screenshot 2023-01-11 at 2 30 53 PM](https://user-images.githubusercontent.com/106531695/211795631-7e6e5d8d-520c-40a4-86f0-b88e4b879cc0.png)